test/integration/v201902-py2/controls/pkgs_spec.rb

```diff
@@ -3,9 +3,18 @@
 version =
   case platform[:family]
   when 'redhat'
-    '2019.2.2-1.el7'
+    case platform[:name]
+    when 'amazon'
+      '2019.2.3-1.el7'
+    when 'centos'
+      '2019.2.3-1.el7'
+    end
+  when 'fedora'
+    '2019.2.0-1.fc30'
+  when 'suse'
+    '2019.2.0-lp151.5.9.1'
```

Comment:
Only the one original line needs to be changed:

```diff
+    '2019.2.3-1.el7'
-    case platform[:name]
-    when 'amazon'
-      '2019.2.3-1.el7'
-    when 'centos'
-      '2019.2.3-1.el7'
-    end
-  when 'fedora'
-    '2019.2.0-1.fc30'
-  when 'suse'
-    '2019.2.0-lp151.5.9.1'
```

Our Fedora and OpenSuse pre-salted images are Python 3 only, so don't need to be tested here (`py2`).